apps/asap-server/src/entities/user.ts

```diff
@@ -196,13 +202,18 @@ export const parseGraphQLUser = (
 
 export const parseUser = (user: RestUser): UserResponse => {
   const teams: UserTeam[] =
-    user.data.teams?.iv?.map(({ id, ...t }) => ({
-      id: id[0],
-      displayName: 'Unknown',
-      ...t,
-      approach: t.approach ? t.approach : undefined,
-      responsibilities: t.responsibilities ? t.responsibilities : undefined,
-    })) || [];
+    user.data.teams?.iv?.map(({ id, ...t }) => {
```

Comment:
why dont you skip the team which does not have the ID instead? maybe log a warning or something too.